utils/data_preprocessing/employee_growth_per_quater_per_domain.py

```diff
@@ -0,0 +1,128 @@
+import sys
+sys.path.append("../..")
```

Comment:
We do not commit this in code. We can use this when we need to do local testing and remove afterwards.